packages/devtools/lib/src/timeline/cpu_flame_chart.dart

```diff
@@ -172,15 +103,19 @@ class CpuFlameChart extends CoreElement {
     }
   }
 
-  Future<void> show() async {
+  void show() async {
```

Comment:
looks like you didn't intend to have this method remain async?

Reply:
whoops. fixed.